main.cpp

```diff
@@ -127,596 +161,485 @@ struct CarWash
-    3) wrap a gift
- */
-
-struct StationersShop //                            1) define an empty struct for each of your 10 types.       
+struct StationersShop      
 {
-    //number of paper sizes for sale                    2) copied and commented-out plain-english property
-    int numPaperSizesForsale = 15; //                   3) member variables with relevant data types.
-    //number of wrapping paper designs for sale     
-    int numWrappingPaperDesignsForsale = 20;     
-    //amount of money spent on electricity per month    
+    int numPaperSizesForsale = 15;    
+    int numWrappingPaperDesignsForsale = 20;        
     float monthElectricityBill = 150.2f;            
-    //amount of profit made per day         
     float profitPerDay = 105.9f;               
-    //number of employees              
     int numberOfEmployees = 10;               
     
-    struct Paper //5)                                 Note that the nested type 'Paper' is related to the 'StationersShop' 
+    struct Paper
     {
-        //2) member variables with relevant data types.  the names are appropriate for the U.D.T.'s purpose.
         bool forPhotoPrinting = false;
         float thicnessInuM = 200.0f;        
         float priceInUsd = 0.1f;
         std::string sizeStandard = "A4";
         std::string manufacturer = "whixe";
 
-        //3) a member function.  it has multiple arguments, some with default values.
-        //the parameter names are related to the work the function will perform.
-        //
         void printDocument(std::string fileName, float zoomPercentage, bool onesided = false); 
-        //Returns the seconds it took to print the photo
         float printPhoto(int qualityIndex);
-        //Returns if theres space on the selling outbox
         bool moveToSellingOutbox(Paper paperB);
         
     };
 
-     //3) a member function.  it has multiple arguments, some with default values.
-    //the parameter names are related to the work the function will perform.
-    //1) returns the amount to be charged in USD
     float makeAPhotocopy(Paper paperA, bool isBlackAndWhite = true , bool onesided = false);
-    //2) returns the amount to be charged in USD
     float takePassportPhoto(Paper paperB, std::string sizeStandard);
-    //3)
     void wrapAGift(std::string wrappingPaperModel); 
 
-
-    //5) a member variable whose type is a UDT.
     Paper paperToBeUsed;  
 };
 
-/*
-Thing 2) Wallet
-5 properties:
-    1) number of slots
-    2) number of credit cards
-    3) amount of cash carried
-    4) amount of weight
-    5) color
-3 things it can do:
-    1) identify the owner
-    2) pay lunch
-    3) enter a building
- */
+float StationersShop::makeAPhotocopy(Paper paperA, bool isBlackAndWhite, bool onesided)
+{    
+    float price = paperA.priceInUsd;
+    if (not isBlackAndWhite)
+    {
+         price = price*2;
+    }
+    if (not onesided)
+    {
+         price = price*2;
+    }
+    return price;
```

Comment:
use `var == false` or `var != true` instead of `not var`
It is more explicit and reads closer to plan English which is what we want our code to eventually be as readable as.
- [ ] add space around the math operators
- [ ] revise